src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystems;
+
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.ctre.phoenix6.hardware.Pigeon2;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class DriveSubsystem extends SubsystemBase {
+    private WPI_TalonSRX leftFrontMotor;
```

Comment:
```suggestion
    private final WPI_TalonSRX leftFrontMotor;
```

Define member variables as `final` when possible to avoid assignment/initialization bugs. Same for all the members below.

Reply:
you missed this comment